devbox/provision/infra/ami.py

```diff
@@ -0,0 +1,28 @@
+import pulumi_aws as aws
+
+import pulumi
+
+
+def get_ami() -> pulumi.Output[aws.ec2.GetAmiResult]:
+    return aws.ec2.get_ami_output(
+        owners=["099720109477"],  # Ubuntu / Canonical
+        filters=[
+            # the version of Ubuntu
+            aws.ec2.GetAmiFilterArgs(
+                name="name",
+                values=["*ubuntu-focal-20.04-amd64-server-20220404"],
```

Comment:
I'll note that we'll probably want to upgrade the version of this on a semi-regular basis. However I'm fine with locking it to a specific AMI here.